package_azureboard_index.json

```diff
@@ -278,6 +278,36 @@
                             "version":  "0.10.0"
                         }
                     ]
+                },
+				{
+                    "name": "MXChip - Microsoft Azure IoT Developer Kit",
+                    "architecture": "stm32f4",
+                    "version": "1.3.1",
+                    "category": "Contributed",
+                    "help":  {
+                        "online": "https://github.com/Microsoft/azure-iot-developer-kit"
+                    },
+                    "url": "https://azureboard2.azureedge.net/prod/arduinopackage/AZ3166-1.3.1.zip",
+                    "archiveFileName": "AZ3166-1.3.1.zip",
+                    "checksum": "MD5:47850b0c88ad774904eeea01c9eecb39",
+                    "size": "6685320",

```

Comment:
the size and checksum is not same the existing package

Reply:
sorry for the mistake, it is updated now.